server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/SearchDataLoader.java

```diff
@@ -127,31 +137,34 @@ SearchData load(SearchWsRequest request) {
     return qualityProfiles.values();
   }
 
-  private List<QProfile> findAllProfiles(DbSession dbSession, SearchWsRequest request) {
+  private List<QProfile> findAllProfiles(DbSession dbSession, SearchWsRequest request, OrganizationDto organization) {
     String language = request.getLanguage();
 
     if (language == null) {
-      return profileLookup.allProfiles(dbSession).stream().filter(qProfile -> languages.get(qProfile.language()) != null).collect(Collectors.toList());
+      return profileLookup.allProfiles(dbSession, organization).stream().filter(qProfile -> languages.get(qProfile.language()) != null).collect(Collectors.toList());
     }
-    return profileLookup.profiles(dbSession, language);
+    return profileLookup.profiles(dbSession, language, organization);
   }
 
-  private Set<String> lookupByProfileName(DbSession dbSession, Map<String, QProfile> qualityProfiles, Set<String> languageKeys, @Nullable String profileName) {
+  private Set<String> lookupByProfileName(DbSession dbSession, OrganizationDto organization, Map<String, QProfile> qualityProfiles, Set<String> languageKeys,
+    @Nullable String profileName) {
     if (languageKeys.isEmpty() || profileName == null) {
       return languageKeys;
     }
 
-    addAllFromDto(qualityProfiles, profileFactory.getByNameAndLanguages(dbSession, profileName, languageKeys));
+    profileFactory.getByNameAndLanguages(dbSession, organization, profileName, languageKeys).forEach(qualityProfile -> qualityProfiles.put(qualityProfile.getLanguage(), qualityProfileDtoToQProfile(qualityProfile, organization)));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 229 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)